Packs/FeedOpenCTI/Integrations/FeedOpenCTI_v4/FeedOpenCTI_v4.py

```diff
@@ -218,8 +232,8 @@ def main():
         max_fetch = arg_to_number(max_fetch)
     else:
         max_fetch = 500
-    start = arg_to_number(args.get('score_start', 1))
-    end = arg_to_number(args.get('score_end', 100)) + 1  # type:ignore
+    start = arg_to_number(params.get('score_start', 1))
+    end = arg_to_number(params.get('score_end', 100)) + 1  # type:ignore
```

Comment:
```suggestion
    start = arg_to_number(args.get('score_start') or params.get('score_start', 1)) 
    end = arg_to_number(args.get('score_end') or params.get('score_end', 100)) + 1  # type:ignore
```

We want to continue supporting the argument, but in the same time consider the params.

Reply:
I think since this is a breaking change anyway, this is a chance to to it the right way.
let me know what you think.